src/main/java/kostka/moviecatalog/service/RatingService.java

```diff
@@ -20,14 +20,15 @@ public RatingService(final DbMovieService dbMovieService) {
     }
 
     public Movie createRating(final RatingDto dto) {
-        Long movieId = dto.getId();
-        if (movieId == null) {
-            LOGGER.error("Dto has no ID specified");
+        Long movieId = dto.getMovieId();
+        int rating = dto.getRating();
+        if (movieId == null || rating < 1 || rating > 10) {
```

Comment:
tu 1 a tu 10 bych nekam extrahoval, kdyz to pouzivas na vice mistech. Nejakou statickou konstantu nebo Utils class, kde muzes dat navic i metodu `isValidRating()` nebo neco podobneho, ktera bude kontrolovat to: `rating < RATING_MIN || rating > RATING_MAX`

Reply:
budu to jeste upravovat tak, ze se bude volat rating mircoservice ktera se mi o to postara, ale jo, dobry point :) díky

Reply:
Jde mi spis hlavne o ty konstanty, ktere jsou v te entite a pak jeste jinde. Kdyz se to (hypoteticky) nekdo rozhodne zmenit, tak stoprocentne zapomene na tu druhou stranu. Proto vzdycky kdyz se pouziva nejaka konstanta dvakrat (a nejlepsi je i kdyz se pouziva jednou) je extrahovat do promenne a tu pouzivat.